src/kuzya.cpp

```diff
@@ -122,14 +122,28 @@ Kuzya::Kuzya(QWidget *parent)
     textEditor = new QsciScintilla(this);
     textEditor->setEolMode(QsciScintilla::EolUnix);
 
+    mOutputTabWidget= new QTabWidget(this);
+
     notificationList = new QListWidget(this);
     notificationList->setObjectName("notificationList");
     notificationList->setVisible(false);
```

Comment:
We should remove all hiding notificationList to be sure, that is won't hide sometime. You might forgot to find from the begin of file